src/containerapp/azext_containerapp/custom.py

```diff
@@ -1996,7 +1996,7 @@ def containerapp_up(cmd,
                     logs_key=None,
                     repo=None,
                     token=None,
-                    branch=None,
+                    branch="main",
```

Comment:
@StrawnSC add some exceptional handling for if the branch info is incorrect or missing, so a user can run the command with --branch (See Jeff's email with more details as well)